maven-compat/src/main/mdo/profiles.mdo

```diff
@@ -308,7 +308,7 @@ under the License.
           <name>checksumPolicy</name>
           <version>1.0.0</version>
           <description>What to do when verification of an artifact checksum fails - warn, fail, etc. Valid values are
-            "fail", "warn" (default) or "ignore"</description>
+            "fail" (default), "warn" or "ignore"</description>
```

Comment:
Given we list all the values, should we really keep the `- warn, fail, etc.` part after the `What to do when verification of an artifact checksum fails` sentence ?
Maybe simply:
`What to do when verification of an artifact checksum fails. Valid values are "fail" (default), "warn" or "ignore".`

Reply:
Correct, avoid duplication.

Reply:
@gnodet 
Thanks for the detail correction.